src/knockout-jqAutocomplete.js

```diff
@@ -94,9 +94,16 @@
         //the binding's update function. keep value in sync with model
         this.update = function(element, valueAccessor) {
             var options = unwrap(valueAccessor()),
-                value = unwrap(options && options.value);
+                value = unwrap(options && options.value),
+                inputProp = options.inputProp || options.labelProp || options.valueProp,
+                valueProp = options.valueProp,
+                sources = ko.unwrap(options.source);
 
```

Comment:
`unwrap` is an alias for `ko.utils.unwrapObservable`, as `ko.unwrap` was not available in KO versions < 3. So, you can use `unwrap` instead.